### PR TITLE
GlobalStateProvider now makes API request to populate calorieCount and waterCount vars

### PR DESCRIPTION
**Pull request by Scott-Guinn**
These variables are now available in the GlobalStateProvider for use in the App along with accompanying setCalorieCount and setWaterCount functions.

### CODE REVIEW

File: components/globalState.js [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
+import axios from 'axios';
 
 export const GlobalStateProvider = (props) => {
+  const [theme, setTheme] = useState('light');
   const [userInfo, setUserInfo] = useState({
     firstName: 'Jane',
     lastName: 'Doe',
     email: 'janeDoe@email.com',
     username: 'username',
+    weightGoal: 150,
```
Comment (emmaknor) [inline components/globalState.js:13]: is there any chance we can add the calorie & water goals to this?

**Review by emmaknor — APPROVED** [review]
Great work! Thank you for setting this up Scott!

File: components/globalState.js (hunk 1 above)
Comment (Scott-Guinn) [reply components/globalState.js:13]: yep good call, doing so now

**Review by taylorsmart — APPROVED** [review]
Not seeing anything of concern in this. 

Dig it - good code.

**Pull request merged**